spec/integration/create_subscriber_list_spec.rb

```diff
@@ -24,7 +24,7 @@
       end
 
       it "creates another subscriber list with a different slug" do
-        create_subscriber_list(title: "oil and gas", tags: { topics: { any: ["oil-and-gas/licensing"] } })
+        create_and_joined_facet_subscriber_list(title: "oil and gas", tags: { topics: { any: ["oil-and-gas/licensing"] } })
```

Comment:
A little confused as to why we're making an AndJoined list out of _just_ 'oil-and-gas' - isn't that one topic?